src/components/Map/MapRenderUtils.tsx

```diff
@@ -60,29 +64,61 @@ export function MapTooltip({ title, properties }: MapTooltipProps): JSX.Element
     ...textStyle,
     marginLeft: theme.spacing(1),
     overflowWrap: 'anywhere',
+    textAlign: newPlantsDashboardEnabled ? 'right' : 'left',
   };
 
   return (
     <>
-      {title && (
-        <Typography fontSize='16px' fontWeight={600} marginBottom={theme.spacing(2)} textAlign='left'>
-          {title}
-        </Typography>
-      )}
-      <table>
-        <tbody>
-          {properties.map((prop: TooltipProperty, index: number) => (
-            <tr key={index}>
-              <td>
-                <Typography sx={keyStyle}>{prop.key}</Typography>
-              </td>
-              <td>
-                <Typography sx={valueStyle}>{prop.value}</Typography>
-              </td>
-            </tr>
-          ))}
-        </tbody>
-      </table>
+      <Box
+        sx={
+          newPlantsDashboardEnabled
+            ? {
+                backgroundColor: theme.palette.TwClrBgSecondary,
+                borderBottom: `1px solid ${theme.palette.TwClrBrdrTertiary}`,
+                padding: theme.spacing(2, 4),
+                borderRadius: 1,
+              }
+            : {}
+        }
+      >
```

Comment:
Passing `undefined` here rather than an empty object feels more clear to me, but it's just a nit. Up to you if you want to include the change.

```suggestion
      <Box
        sx={
          newPlantsDashboardEnabled
            ? {
                backgroundColor: theme.palette.TwClrBgSecondary,
                borderBottom: `1px solid ${theme.palette.TwClrBrdrTertiary}`,
                padding: theme.spacing(2, 4),
                borderRadius: 1,
              }
            : undefined
        }
      >
```